tests/parsing/__snapshots__/test_property.ambr

```diff
@@ -0,0 +1,99 @@
+# serializer version: 1
+# name: test_from_ics[attendee]
+  list([
+    ParsedProperty(name='begin', value='VEVENT', params=None),
+    ParsedProperty(name='attendee', value='mailto: jsmith@example.com', params=[ParsedPropertyParameter(name='RSVP', values=['TRUE']), ParsedPropertyParameter(name='ROLE', values=['REQ-PARTICIPANT'])]),
+    ParsedProperty(name='attendee', value='jdoe@example.com","mailto:jqpublic @example.com":mailto:jsmith@example.com', params=[ParsedPropertyParameter(name='DELEGATED-TO', values=['"mailto'])]),
+    ParsedProperty(name='end', value='VEVENT', params=None),
```

Comment:
By the way, this snapshot contains examples where this heuristic does not parse content correctly, like the grammar does. This is an example of the type of bug that most other icalendar libraries have when implementing the spec, which motivated using pyparsing with a real grammar.

The "bugs" are recorded here and can be spotted when running tests that compare both parsing approaches.